README.md

```diff
@@ -1091,13 +1091,13 @@ You can configure yt-dlp by placing any supported command line option to a confi
 
 1. **Main Configuration**:
     * The file given by `--config-location`
-1. **Portable Configuration**: (Recommended for portable installations)
+2. **Portable Configuration**: (Recommended for portable installations)
     * If using a binary, `yt-dlp.conf` in the same directory as the binary
     * If running from source-code, `yt-dlp.conf` in the parent directory of `yt_dlp`
-1. **Home Configuration**:
+3. **Home Configuration**:
     * `yt-dlp.conf` in the home path given by `-P`
     * If `-P` is not given, the current directory is searched
-1. **User Configuration**:
+4. **User Configuration**:
```

Comment:
Using 1. everywhere is intentional. Pls revert

Reply:
sorry, my bad